languoids/tree/grea1241/md.ini

```diff
@@ -6,9 +6,10 @@ level = family
 [classification]
 sub = **hh:hv:Manoharan:Andamanese**
 familyrefs = 
-	**hh:hv:Abbi:Andamanese**
+	**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
 subrefs = 
 	**hh:hv:Manoharan:Andamanese**
+family = 
```

Comment:
I tried to remove fields with empty values. Did you add this manually or did it sneak in as serialization of the classification object? If the latter, I'll have to fix this.